<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 221, 221);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="visor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>400</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="original">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: red;
font: 75 48pt &quot;Times New Roman&quot;;
 border-style: solid;
 border-width:1px;
 border-radius:25px;
 border-color: red;
 max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregarfoto">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: red;
font: 75 36pt &quot;Times New Roman&quot;;
 border-style: solid;
 border-width:1px;
 border-radius:25px;
 border-color: red;
 max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Gestionar comprobantes de pago</string>
    </property>
   </widget>
   <widget class="QLabel" name="visor_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>400</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Abrir</string>
    </property>
    <addaction name="actionFoto"/>
    <addaction name="actionCerrar"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFoto">
   <property name="text">
    <string>Foto</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>original</sender>
   <signal>clicked()</signal>
   <receiver>visor</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>original</sender>
   <signal>clicked()</signal>
   <receiver>visor_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
